src/core/config/platform.h

```diff
@@ -107,4 +107,15 @@
 #define OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_PLATFORM_RADIO_SPINEL_RX_FRAME_BUFFER_SIZE
+ *
+ * Specifies the rx frame buffer size used by `SpinelInterface` in RCP host (posix) code. This is applicable/used when
+ * `RadioSpinel` platform is used.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_PLATFORM_RADIO_SPINEL_RX_FRAME_BUFFER_SIZE
+#define OPENTHREAD_CONFIG_PLATFORM_RADIO_SPINEL_RX_FRAME_BUFFER_SIZE 2048
```

Comment:
Should we increase the default size given that we are already seeing issues with 2048?

Reply:
I was considering it as well. I agree, it may be good to change the default to something larger (specially since this is intended for RCP host posix which would not be as memory constrained)

What would be good default number? `4096`?

Reply:
Went for an 8K default buffer in new push.